lib/qesapdeployment.pm

```diff
@@ -874,6 +874,7 @@ sub qesap_upload_crm_report {
         provider => $prov,
         filter => $host,
         host_keys_check => 1,
+        failok => 1,
```

Comment:
I think it might be good to have something like softfail here that will tell us about 'crm report' failing and it needs to be fixed. Maybe at the end of the test there could be a curl call to check if file is present in the results?
Something like checking  url 'https://<openqa instance>/tests/<test id>/logfile?filename=filename'
@mpagot wdyt?

Reply:
The point here is that we are sometime calling `qesap_upload_crm_report` after an Ansible failure. The failure could be so early that the cluster is not there at all. 
I'd propose not to add a softfailure but to  add a `failok` argument in the `qesap_upload_crm_report` too to let the caller to decide. I propose to do it later in a different PR. I'm also ok to fully remove this `failok` change from this PR at all and address everything in a new dedicated PR.

Reply:
I agree with Michele. @rjagu1 if you want to go for it in this ticket, it's as simple as adding

**qesapdeployment.pm, line 903:**
`my $failok = $args{failok} // 0;` (or 1, to have failing be ok be default)

and then on the subsequent `qesap_ansible_cmd` and `qesap_ansible_fetch_file` on lines 906 and 911, add:
`failok => $failok`